<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetScent</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 43, 48);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 43, 48);
color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>813</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 57, 62);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="captureTab">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(54, 57, 62);
color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="title">
        <string>Capture</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0" colspan="7">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>530</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>528</width>
               <height>18</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout"/>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="maximumSize">
             <size>
              <width>275</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="CustomButton" name="stopButton">
          <property name="minimumSize">
           <size>
            <width>98</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>98</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(87, 227, 137);
color: rgb(23, 23, 23);</string>
          </property>
          <property name="text">
           <string>Stop Capture</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="CustomButton" name="clearButton">
          <property name="minimumSize">
           <size>
            <width>49</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>49</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(87, 227, 137);
color: rgb(23, 23, 23);</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="statusLabel">
          <property name="minimumSize">
           <size>
            <width>69</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>69</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="CustomButton" name="startButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(87, 227, 137);
color: rgb(23, 23, 23);</string>
          </property>
          <property name="text">
           <string>Start Capture</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="ComboCheckBox" name="filterBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QLabel" name="apiSettingLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(54, 57, 62);</string>
        </property>
        <property name="text">
         <string>Geolocation API Key:</string>
        </property>
       </widget>
       <widget class="QLabel" name="apiLinkLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Hint: https://ipgeolocation.io/</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="apiKeyText">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="setApiKeyButton">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>89</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Key</string>
        </property>
       </widget>
       <widget class="QLabel" name="keyDetectedLabel">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>30</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 97, 81);</string>
        </property>
        <property name="text">
         <string> No key detected</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomButton</class>
   <extends>QPushButton</extends>
   <header>CustomButton.h</header>
  </customwidget>
  <customwidget>
   <class>ComboCheckBox</class>
   <extends>QComboBox</extends>
   <header>ComboCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
